scripts/bench_validate_bacnet_vs_niagara.py

```
@@ -0,0 +1,75 @@
+
+    if args.api:
+        base = args.api.rstrip("/")
+        body = {"write_report": args.write_report, "report_label": "cli"}
+        if args.config:
+            body["config_path"] = args.config
+        req = urllib.request.Request(
+            f"{base}/api/bench/validate/bacnet-vs-niagara",
+            data=json.dumps(body).encode("utf-8"),
+            headers={"Content-Type": "application/json"},
+            method="POST",
+        )
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)